projects/ngx-property-grid/src/lib/dynamic-component-load.directive.ts

```diff
@@ -62,7 +61,7 @@ export class DynamicComponentLoadDirective implements OnInit, OnDestroy, OnChang
   }
 
   private initComponent(component: ComponentRef<PropertyValueAccess>) {
-    component.instance.value = this.propertyValue.value;
+    component.instance.value = this.options[this.meta.key]; 
```

Comment:
why not use `PropertyValue`?

Reply:
This is part of the second change - specifying meta data dynamically (Which works, but I'd not intended this one to be merged just yet).

But this solved an angular refresh issue
    So when specifying meta data dynamically, after a change to a value in the property grid, it re-queries for new meta data.
    However when this happens the dynamically created angular controls are not refreshing (I think as its loosing the change tracking in the PropertyValue object), by applying these values directly to the dynamically created angular components it solved the refresh issue.

Reply:
I recomend to used `PropertyValue` instead. Because the custom controll will use `p.value` to modify values.

https://github.com/mokeyish/ngx-property-grid/blob/e92c040bb37f2a5bd44dd3ebd225207d48898331/src/app/showcase/app.component.html#L20